app/scripts/models/auth_brokers/fx-ios-v2.js

```diff
@@ -0,0 +1,46 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * The auth broker to coordinate authenticating for Sync when
+ * embedded in the Firefox for iOS 2.0.
+ */
+
+define([
+  'lib/auth-errors',
+  'models/auth_brokers/fx-desktop-v1'
+], function (AuthErrors, FxDesktopV1AuthenticationBroker) {
+  'use strict';
+
+  var proto = FxDesktopV1AuthenticationBroker.prototype;
+
+  var FxiOSV2AuthenticationBroker = FxDesktopV1AuthenticationBroker.extend({
+    type: 'fx-ios-v2',
+
+    commands: _.extend({}, proto.commands, {
+      SYNC_PREFERENCES: 'fxaccounts:sync_preferences'
```

Comment:
The original desktop commands all lack the `fxaccounts:` prefix, but adding the prefix seems more future proof for when Fx iOS converts to WebChannels.